output/md/md.go

```diff
@@ -730,3 +730,8 @@ func (m *Md) addNumberToTable(data [][]string) [][]string {
 
 	return data
 }
+
+// replace space by %20 in markdown links (https://spec.commonmark.org/0.30/#link-destination)
+func normalizeMdLink(name string) string {
+	return strings.Replace(name, " ", "%20", -1)
```

Comment:
@YauhenPylAurea Thank you for your commit!

Looking at https://spec.commonmark.org/0.30/#link-destination, it looks like there is a little more implementation needed.

Is it possible to add a few more implementations and tests?